charts/covidDataExplorer/CovidChartUrl.ts

```diff
@@ -58,20 +73,19 @@ export class CovidQueryParams {
             "testsPerCaseMetric",
             "positiveTestRate"
         ]
-        const timelines = ["totalFreq", 7, "dailyFreq"]
         const perCapita = [true, false]
         const aligned = [true, false]
         const yScale = ["log", "linear"] // todo
         const tab = ["map", "chart"] // todo
         const combos: any = []
         metrics.forEach(metric => {
-            timelines.forEach(timeline => {
+            intervalOptions.forEach(interval => {
```

Comment:
Nice to get rid of this "timeline" naming collision! Now we just need to resolve our two `Tables`... 😮

Reply:
Where do you see two tables? I count at least 4 😮